src/components/filterbox.jsx

```diff
@@ -151,21 +153,20 @@ function useSearch(model, selfRef, inputRef) {
   return { onSearch };
 }
 
-export default function Filterbox({ model, layout }) {
+export default function Filterbox({ model, layout, showFilterbox }) {
```

Comment:
Why does the filterbox take a `showFilterbox` arg? Shouldn't this be controlled in the parent container and never render a filterbox then?